### ISSUE TITLE
Binary compatibility since grails 3.1.4

### ISSUE THREAD

**Issue by sgaide** [agorapulse/grails-newrelic#1]
Hi guys,

it seems that since grails 3.1.4 the newrelic plugin is no more binary compatible.
if used without rebuilding it I get this error when starting my app :

```
[...]

Caused by: org.springframework.beans.PropertyBatchUpdateException: Failed properties: Property 'grailsApplication' threw exception; nested exception is groovy.lang.MissingMethodException: No signature of method: static grails.artefact.Interceptor$Trait$Helper.setGrailsApplication() is applicable for argument types: (grails.plugin.newrelic.NewRelicInterceptor, grails.core.DefaultGrailsApplication) values: [grails.plugin.newrelic.NewRelicInterceptor@1124dd79, grails.core.DefaultGrailsApplication@da28d03]
```

if i rebuild the plugin, after making it use grails 3.1.4, and use this new build in my app, everything is perfect.

i would much prefer use the plugin `as is`.

thanks a lot,

Sebastien

**Comment by sgaide** [comment]
may be linked to [this grails issue](https://github.com/grails/grails-core/issues/9793) and its correction

**Comment by benorama** [comment]
Thanks, I'll check that tomorrow.

**Comment by sgaide** [comment]
you are welcome, and thanks to you for this great plugin !

**Comment by ghost** [comment]
Yes, I got the same issue trying to deploy a 3.0.15 to AWS. Hopefully 3.0.16 fixes this.

**Comment by benorama** [comment]
I've release version 3.27.0 version with Grails 3.1.4.
I've also released a 3.26.1.1 version 2 weeks ago to solve this issue.
So it should be solved now.

**Comment by sgaide** [comment]
Yes, thanks Benoit, the 3.26.1.1 solved the problem !
